PhotoKingdomAPI/PhotoKingdomAPI/Controllers/QueuesController.cs

```diff
@@ -11,6 +11,31 @@ public class QueuesController : ApiController
     {
         private Manager m = new Manager();
 
+        // GET: api/Queues
+        public IHttpActionResult Get()
+        {
+            return Ok(m.QueueGetAll());
+        }
+
+        // GET: api/Queues/5
+        public IHttpActionResult Get(int? id)
```

Comment:
This URL structure doesn't follow the Microsoft API convention. Really, it should be under api/attractions/5/queues , because u want to get the queues collection that belongs to attraction #5 . But it's okay I'll merge it because it still works fine. But it could be confusing to others because normally api/queues/5 means fetch the queue with ID #5.